src/main/java/com/transportcompany/transportcompany/models/entities/User.java

```diff
@@ -2,19 +2,25 @@
 
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 import jakarta.persistence.*;
+import jakarta.validation.constraints.NotNull;
+import lombok.AllArgsConstructor;
 import lombok.Data;
+import lombok.NoArgsConstructor;
 
 import java.util.Optional;
 
 @Data
 @Entity
-@Table(name ="user")
+@AllArgsConstructor
+@NoArgsConstructor
+@Table(name ="user_table")
 @JsonIgnoreProperties({"hibernateLazyInitializer", "handler"})
-public class User {
 
```

Comment:
As GitHub does the code formatting I can't tell if between @JsonIgnoreProperties({"hibernateLazyInitializer", "handler"}) annotation and User class is an empty row, but if exists, it should be deleted.